types.d.ts

```diff
@@ -26,19 +26,32 @@ declare module "@nice-digital/global-nav" {
 		| "cks"
 		| "journals";
 
+	type Link = {
+		text: string;
+		url: string;
+	};
+
 	type OnNavigatingCallback = (
 		e: { element: HTMLAnchorElement; href: string }
 	) => void;
 
+	export type IdamProviderProps = {
+		provider: "idam";
+		links: Array<Link>;
+		displayName: string;
```

Comment:
Shouldn't this be optional? Or nullable?
```suggestion
		displayName?: string;
```